Sources/BluetoothServices/Characteristics/MeasurementInterval.swift

```diff
@@ -0,0 +1,58 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import NIO
+import SpeziBluetooth
+
+
+/// Represents the time between measurements.
+public enum MeasurementInterval {
```

Comment:
If this is from some sort of standard it would be gerat if we could link to it in the documentation.